src/test/README.md

```diff
@@ -38,7 +36,7 @@ the `src/qt/test/test_main.cpp` file.
 For example, to run just the `getarg_tests` suite of tests:
 
 ```bash
-test_bitcoin --log_level=all --run_test=getarg_tests
+build/src/test/test_bitcoin --log_level=all --run_test=getarg_tests
```

Comment:
The previous variant assumed `.` is in `PATH`!

Reply:
I couldn't find that in the docs, do most people have that set?